serveradmin/servershell/static/js/servershell/result.js

```diff
@@ -279,7 +282,7 @@ register_inline_editing = function(cell) {
 
         cell.html(content);
         cell.append(button);
-        cell.append(`<div><b>${attribute.regex}</b></div>`);
+        cell.append(`<div><b>${attribute.regex !== null ? attribute.regex : 'No Regexp'}</b></div>`);
```

Comment:
Is that safe in any cases ? I am checking explicit for null because it might be undefined, false or something else.

Reply:
Ah, ok. If it's really explicit null, then no. It would also react to `undefined` and `false` mostly too. I did some quick tries:
``` js
let foo = '0';
foo || 'something'  // "0"

let foo = undefined;
foo || 'something'  // "something"

let foo = false;
foo || 'something'  // "something"

let foo = null;
foo || 'something'  // "something"

let foo = 0;
foo || 'something'  // "something"
```

So, might actually be more than you want. Then forget what I sent. ^^